src/signed.rs

```diff
@@ -59,6 +59,13 @@ pub struct UncheckedSigned<T: Signable, S> {
     signature: S,
 }
 
+impl<T: Signable, S: Signature> UncheckedSigned<T, S> {
+    #[cfg(test)]
```

Comment:
I think you can slap this on even one line higher.

Reply:
Nope, fmt doesn't allow that.